### PR TITLE
Fair Visualization release 1 - additional changes

### PR DESCRIPTION
**Pull request by bklaing2**
## Purpose

- Moves the Cite As section to the left sidebar
- MetadaDisplay for DOIs now shows creators and contributors as tables
- Updated the look of the title for DOIs

closes: _Add github issue that originated this PR_

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: src/pages/doi.org/[...doi].tsx [hunk 1]
```diff
@@ -642,16 +645,25 @@ const WorkPage: React.FunctionComponent<Props> = ({ doi, metadata }) => {
         </Modal>
       </>)
     }
+    // const handleUrl =
```
Comment (jrhoads) [inline src/pages/doi.org/[...doi].tsx:648]: I would delete the commented code.

File: src/pages/doi.org/[...doi].tsx [hunk 2]
```diff
@@ -642,16 +645,25 @@ const WorkPage: React.FunctionComponent<Props> = ({ doi, metadata }) => {
...
     return (
       <>
         <Col md={3} className="panel-list" id="side-bar">
           {downloadMetadataButton()}
+          <CiteAs doi={work} />
           <ShareLinks url={'doi.org/' + work.doi} title={work.titles[0] ? work.titles[0].title : undefined} />
         </Col>
         <Col md={9} className="panel-list" id="content">
-          <Work doi={work}></Work>
+            {/* <Title title={ReactHtmlParser(titleHtml)} url={handleUrl} link={'https://doi.org/' + work.doi} /> */}
```
Comment (jrhoads) [inline src/pages/doi.org/[...doi].tsx:664]: Commented code

**Review by jrhoads — CHANGES_REQUESTED** [review]
Looks good.  I would say it's good to go after we delete the commented code.

File: src/pages/doi.org/[...doi].tsx (hunk 1 above)
Comment (bklaing2) [reply src/pages/doi.org/[...doi].tsx]: Deleted this

Comment (bklaing2) [reply src/pages/doi.org/[...doi].tsx]: Deleted this

File: src/pages/doi.org/[...doi].tsx (hunk 2 above)
Comment (bklaing2) [reply src/pages/doi.org/[...doi].tsx]: Deleted this

**Pull request merged**